README.md

```diff
@@ -189,6 +189,149 @@ We only did a small refactor, so the build tab running `npm run build` should
 show no errors, and the local page under http://localhost:8080/ should look like
 it did before starting this step.
 
+## Step 3: Modeling data
+
+To build our GitHub activity feed reader, we'll eventually fetch data from this
+endpoint: [https://gh-feed.vercel.app/api](https://gh-feed.vercel.app/api).
+
+This endpoint doesn’t require API keys and works as follows:
+- It accepts two query parameters: `user` (a `string`) and `page` (an `int`).
+  For example: `https://gh-feed.vercel.app/api?user=jchavarri&page=1`
+- It returns an object (let's call it `feed`) with a key `entries`, which is an
+  array of feed entries. Each entry has the following structure:
+  - **content**: The content of the entry, usually HTML as a string.
+  - **links**: An array of associated links, each with:
+    - **href**: The URL the link points to (`string`).
+    - **title**: The link title or description (`string`).
+  - **title**: The entry title (`string`).
+  - **updated**: A timestamp of the last update (`float`).
+
+> [!NOTE]
+> 
+> There are other keys in the returned data, but we will ignore them for
+> simplicity.
+
+In this step, we’ll define some OCaml types to represent this data in our
+application. We’ll use records for the objects and lists for the arrays. Refer
```

Comment:
While I was following along, I just made these types:

```reason
type link = {
  href: string,
  title: string,
};

type entry = {
  content: option(string),
  links: array(link),
  title: string,
  updated: float,
};

type feed = {entries: array(entry)};
```
I didn't notice that I was supposed to use lists until I got an error at the decode part.

Reply:
The docs mentioned:

> We’ll use records for the objects and lists for the arrays.

But I agree it's better to use arrays for now, and introduce lists later (maybe at the end of the workshop? once some UI has been done). Fixed in https://github.com/ahrefs/react-alicante-workshop/pull/3/commits/49637671ffc4cca25e31186069f0c6bb55a1ab9b.